aem65/src/test/resources/component-configs/core-components/teaser/descriptionFromLinkedPage.yaml

```diff
@@ -0,0 +1,17 @@
+Link & Actions:
```

Comment:
Instead of configuring multiple dialog fields in one test I suggest only to configure the verified fields, and the rest have preconfigured and created by Sling. So you then have multiple components with different configs delivered by Sling and only choose which component to work on during a test case - much faster than configuring everything with Bobcat in the UI

Reply:
done - I extended content on test page